docs/source/config_file.rst

```diff
@@ -816,6 +816,19 @@ These options may only be set in the global section (``[mypy]``).
 
     Show absolute paths to files.
 
+.. confval:: force_lowercase_builtins
+
+    :type: boolean
+    :default: False
+
+    Force to use ``List`` instead of ``list`` .
```

Comment:
Same comment as with `force_union_syntax` -- it looks like this option just impacts whether or not the messages mypy displays to the user use PEP-585 syntax. Also: the config option is `force_uppercase_builtins`, not `force_lowercase_builtins`

Reply:
Yes, this is just about messages. This is why I put these two in "Configuring error messages" category. Is there any better way to describe them?

Reply:
> This is why I put these two in "Configuring error messages" category.

Ah, I didn't spot that, sorry!



> Is there any better way to describe them?

I think "in error messages" is a useful qualifier -- have made some suggestions in review comments 👍